DigitalLearningSolutions.Web/Helpers/CustomPromptHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.ViewModels.MyAccount;
+
+    public class CustomPromptHelper
+    {
+        private readonly ICustomPromptsService customPromptsService;
+
+        public CustomPromptHelper(ICustomPromptsService customPromptsService)
+        {
+            this.customPromptsService = customPromptsService;
+        }
+
+        public List<EditCustomFieldViewModel> GetCustomFieldViewModelsForCentre(
+            int centreId,
+            string? answer1,
+            string? answer2,
+            string? answer3,
+            string? answer4,
+            string? answer5,
+            string? answer6)

```

Comment:
Could we pass in delegateUser rather than all the answers here and then do the mapping to answers on L26?

Reply:
Ignore this - just noticed that it's used with model too! 👍